DataFormats/FTLDigi/interface/FTLDigiCollections.h

```diff
@@ -11,13 +11,30 @@
 #include "DataFormats/FTLDigi/interface/BTLSample.h"
 #include "DataFormats/FTLDigi/interface/ETLSample.h"
 
-typedef FTLDataFrameT<FastTimeDetId,FTLSample> FTLDataFrame;
+namespace mtdhelpers {
+  struct FTLRowColDecode {
+    static inline int row(const DetId& id, const std::vector<FTLSample>& data) { return -1; } // no rows or columns
+    static inline int col(const DetId& id, const std::vector<FTLSample>& data) { return -1; }
+  };
+
+  struct BTLRowColDecode {
+    static inline int row(const DetId& id, const std::vector<BTLSample>& data) { return data.front().row(); }
```

Comment:
why do these functions need the DetId?

Reply:
They used to need it before I moved the row/column for BTL into the digi. I'll clean it up.